doc/rg.1.md

```diff
@@ -278,6 +278,11 @@ Project home page: https://github.com/BurntSushi/ripgrep
 : Print only the matched (non-empty) parts of a matching line, with each such
   part on a separate output line.
 
+--passthru, --passthrough
+: Show both matching and non-matching lines. This is equivalent to adding ^ to
+  the list of search patterns. This option overrides --count and cannot be used
+  with --only-matching or --replace.
```

Comment:
Yeah I agree that we shouldn't mention `^`. I could see the implementation changing for example to support `--replace` by implementing this flag differently. (e.g., By not modifying the regex.)